src/Mod/Sketcher/Gui/ViewProviderSketch.h

```diff
@@ -523,6 +527,9 @@ class SketcherGuiExport ViewProviderSketch : public PartGui::ViewProvider2DObjec
     virtual bool mouseButtonPressed(int Button, bool pressed, const SbVec2s& cursorPos, const Gui::View3DInventorViewer* viewer) override;
     //@}
 
+    /// moves a selected constraint

```

Comment:
See below for the removal of access specifier

Reply:
Yes it's because I want DrawSketchHandler onMouseMove to move the constraint. I don't know what is an attorney and encapsulation doesn't ring a bell either.

Reply:
Let's start here:
https://forum.freecadweb.org/viewtopic.php?style=3&f=10&t=66095